### ISSUE TITLE
Eliminate alt labels in NMP FPs

### ISSUE THREAD

**Issue by michihdeu** [TravelMapping/DataProcessing#417]
Maybe I asked before.... dunno....

Is it really necessary having alt labels in NMP FP entries?

https://github.com/TravelMapping/HighwayData/pull/4579/commits/3ef475d58a555c92f8766f8f3ac9917f98909c80

If it is not necessary, we might change it one day.

**Comment by yakra** [comment]
I support removing them.
If implemented, I'll change existing FP entries to remove alts.
@jteresco, opinions?

**Comment by jteresco** [comment]
I'm not coming up with any reason to include them.  When labels change, the NMP FP entries would have to change either way.

**Comment by michihdeu** [comment]
I have just **removed unused alt labels**. That can always happen. It is a hassle that the FP entry is no longer valid for this reason. 

Since there seems to be no need for them, I'll change the title.

**Comment by michihdeu** [comment]
- [ ] Eliminate alt labels from nmp files
- [ ] Eliminate alt labels from [nearmisspoints.log](https://travelmapping.net/logs/nearmisspoints.log)
- [ ] Eliminate alt lables from [nmpfps.log](https://github.com/TravelMapping/HighwayData/blob/master/nmpfps.log)

**Comment by yakra** [comment]
> * [ ] Eliminate alt labels from nmp files

You mean tm-master.nmp, and the per-region .nmp files, right? This can be checked already.

**Comment by michihdeu** [comment]
yep. my bad. Thanks.

**Comment by yakra** [comment]
The only potential sticking point I see is that nearmisspoints.log uses `str(Waypoint)`. This is the only production code that uses it, with some scattered debug text commented out. Jim, do you see any pressing need to keep `" [alt: " + str(self.alt_labels) + "]"` in `Waypoint.__str__`? OK to go ahead & trim it out?

**Comment by jteresco** [comment]
It's been a long time since I first got NMP stuff going.  I think if we're going to remove them from nmpfps.log, we should remove them from nearmisspoints.log, as that's where they're copied from.  Seems like it's good to make the change in `Waypoint.__str__`.

**Comment by yakra** [comment]
With the changes to siteupdate and nmpfps.log, we end up matching a few more FP entries:
```diff
yakra@epoch:~/ytm/DataProcessing/siteupdate/cplusplus$ diff ST_dl/logs/nmpfpsunmatched.log ST_alt/logs/nmpfpsunmatched.log
23,24d22
< eng.a0061 B6428_E [alt: ['+B6428']] (53.590283,-1.495187) NMP eng.a0061 B6428_W (53.590566,-1.495568) eng.b6428 A61_N (53.590566,-1.495568)
< eng.a0061 B6428_W (53.590566,-1.495568) NMP eng.a0061 B6428_E [alt: ['+B6428']] (53.590283,-1.495187) eng.b6428 A61_S (53.590283,-1.495187)
56d53
< eng.b6428 A61_N (53.590566,-1.495568) NMP eng.a0061 B6428_E [alt: ['+B6428']] (53.590283,-1.495187) eng.b6428 A61_S (53.590283,-1.495187)
68c65
< esppv.ap001 ESP/FRA [alt: ['+FRA/ESP']] (43.339904,-1.752577) NMP fra.a063 1 (43.339927,-1.752169) fra.e5 1(A63) (43.339927,-1.752169) fra.e70 1(A63) (43.339927,-1.752169) fra.e80 1(A63) (43.339927,-1.752169)
---
> esppv.ap001 ESP/FRA (43.339904,-1.752577) NMP fra.a063 1 (43.339927,-1.752169) fra.e5 1(A63) (43.339927,-1.752169) fra.e70 1(A63) (43.339927,-1.752169) fra.e80 1(A63) (43.339927,-1.752169)
90d86
< fraidf.e15 D7 [alt: ['+4(BlvdPer)']] (48.816161,2.360365) NMP fraidf.d000794 Par/VdM (48.816024,2.360435)
92,93d87
< fraidf.e5 PtePla [alt: ['+10(BlvdPer)']] (48.827435,2.292452) NMP fraidf.d0061a92 Par/HdS (48.827119,2.292312)
< fraidf.e50 D7 [alt: ['+4(BlvdPer)']] (48.816161,2.360365) NMP fraidf.d000794 Par/VdM (48.816024,2.360435)
155,156d148
< md.md005 MD373_S (38.700964,-76.875651) NMP md.md373 MD5_W [alt: ['+MD5', '+BraRd']] (38.700834,-76.875973)
< md.md373 MD5_W [alt: ['+MD5', '+BraRd']] (38.700834,-76.875973) NMP md.md005 MD373_S (38.700964,-76.875651)
180,181d171
< wv.us035 CR17/5 [alt: ['+OldUS35', '+WV817_HenN']] (38.818217,-82.117116) NMP wv.wv817 CR17/5 (38.818696,-82.116752)
< wv.wv817 CR17/5 (38.818696,-82.116752) NMP wv.us035 CR17/5 [alt: ['+OldUS35', '+WV817_HenN']] (38.818217,-82.117116)
```
esppv.ap001 ESP/FRA is still unmatched because it's a holdover from before the FRA split, and doesn't refer to the French subregions.
All the rest don't match now because some or all of the alt labels listed have been removed. Take out the alt labels, and they match again!

**Comment by yakra** [comment]
closed in #430

**Comment by michihdeu** [comment]
Thx for the implemenation 😃
